packages/next/taskfile.js

```diff
@@ -2720,6 +2720,12 @@ export async function check_error_codes(task, opts) {
     })
   } catch (err) {
     if (process.env.CI) {
```

Comment:
Why do we only fail in CI? By failing locally, you avoid one roundtrip to CI because you get the failure early. That way you also avoid the potential confusion when local tests fail due to missing error codes.

Reply:
This code is inside `pnpm build`, so removing the check would fail that command instead of updating `errors.json`. The update is done by the command afterwards -> https://github.com/vercel/next.js/pull/73807/files#diff-9d85a2d25346a67c95d98aa8d81b4d677b90708bdc40e03488ce0586edaade7eR2731